src/lib/y2storage/device.rb

```diff
@@ -111,6 +111,16 @@ class Device
     #   @return [Boolean]
     storage_forward :exists_in_staging?
 
+    # @!method detect_resize_info
+    #   Information about the possibility of resizing a given device.
+    #   If the device has any children, they are also taken into account;
+    #   the result of this method is the combined information about this device
+    #   and all its children.
+    #   @see can_resize?
+    #
+    #   @return [ResizeInfo]
+    storage_forward :detect_resize_info
```

Comment:
I believe this should be:
```ruby
storage_forward :detect_resize_info, as: "ResizeInfo"
```
So the method retains the old behavior of returning a `Y2Storage::ResizeInfo` instead of a `Storage::ResizeInfo`

Reply:
ok